src/lib/FormattedInput.tsx

```diff
@@ -79,11 +80,19 @@ const FormattedInput = <T extends FieldValues>(props: FormattedInputProps<T>) =>
                   }}
                   onFocus={focusHandler}
                   style={style}
+                  className={`form-control ${className}`}
```

Comment:
@severinoAmmirati 

Can you please add the `className` as an argument to `classnames` on line 48. Than you can revert the changes here and for the PatternFormat 

Otherwise the invalid class doesn't get rendered.

Reply:
@neotrow Updated as requested